src/main/java/com/ibm/watson/modelmesh/payload/LoggingPayloadProcessor.java

```diff
@@ -31,6 +33,11 @@ public String getName() {
     @Override
     public boolean process(Payload payload) {
         LOG.info("Payload: {}", payload);
-        return true;
+        return false;
+    }
+
+    @Override
+    public void close() throws IOException {
+        // do nothing
     }
```

Comment:
Nit: could include a `default` empty implementation of this in the `PayloadProcessor` interface which would avoid having it in most of the impls.

Reply:
good point, will do.